literalai/prompt_engineering/prompt.py

```diff
@@ -214,9 +214,9 @@ def to_langchain_chat_prompt_template(self, additional_messages=[]):
 
         class CustomChatPromptTemplate(ChatPromptTemplate):
             orig_messages: Optional[List[GenerationMessage]] = Field(
-                default_factory=list
```

Comment:
they both do the same thing I would keep list though (it seems easier to read)

Reply:
No they do not as per the linter error: 
```
literalai/observability/generation.py:75: error: Argument "default_factory" to "Field" has incompatible type "type[dict[Any, Any]]"; expected "Union[Callable[[], Never], Callable[[dict[str, Any]], Never]]"  [arg-type]
```